rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -754,13 +796,19 @@ def train(
                     self._set_lm_features(batch_docs[index], ex, attribute)
                 batch_start_index += batch_size
 
-    def process(self, message: Message, **kwargs: Any) -> None:
-        """Process an incoming message by computing its tokens and dense features.
+    def process(self, messages: List[Message]) -> None:
```

Comment:
Are we happy with the name `process`? There is also `process_training_data`. Maybe we want to call this `process_messages`?

Reply:
Let's stay consistent with the other ones and keep this `process` for now. Refactoring and improving naming is a great thing to do but we should focus on the migration itself now and don't do too many things at once.